core/src/main/java/org/kiwix/kiwixmobile/core/main/AddNoteDialog.kt

```diff
@@ -399,7 +398,7 @@ class AddNoteDialog : DialogFragment() {
 
   companion object {
     @JvmField val NOTES_DIRECTORY =
-      Environment.getExternalStorageDirectory().toString() + "/Kiwix/Notes/"
+      instance.getExternalFilesDir("").toString() + "/Kiwix/Notes/"
     const val TAG = "AddNoteDialog"
```

Comment:
`Environment.getExternalStorageDirectory()` returns the primary/external storage **directory**. It can be build-in storage in a device. `context.getExternalFilesDir("")` returns absolute paths to application-specific directories on all shared/external **storage devices**. Do we need to store this directory in external storage devices? It can be ejected by the user.

Reply:
hi @gouri-panda ,

It will save on app specific directory , I changed this method because ```Environment.getExternalStorageDirectory()``` is depreciated.

https://developer.android.com/reference/android/os/Environment#getExternalStorageDirectory()